src/Components/Directory.Build.targets

```diff
@@ -20,6 +22,15 @@
           PackagePath="ConfigurationSchema.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- ensure the config generator is built -->
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\Tools\ConfigurationSchemaGenerator\ConfigurationSchemaGenerator.csproj"
+                      SetTargetFramework="TargetFramework=$(NetCurrent)"
+                      OutputItemType="_ReferencedExecutable"
```

Comment:
Can you explain what does this mean or where is it used? I'm missing where you are using it.

Reply:
Hmm, I must have copied this from somewhere, but I can't find where anymore. Removed.